services/KeyVault/vaults/alerts.yaml

```diff
@@ -165,63 +165,6 @@
     properties:
       scope: Resource
       multiResource: true
```

Comment:
@JoeyBarnes IMO, we should remove the conflict here and go with what the ALZ team has suggested. Thoughts?